### DISCUSSION TITLE
Where to find a list of plugins?

### DISCUSSION THREAD

**Discussion by 8ctopus** [eslint/eslint#15449] [Q&A] [answered]
Hello everyone, just started using eslint and I'm wondering where I can find a list of all plugins? I expected to find it [here](https://eslint.org/docs/user-guide/configuring/plugins) but without luck. I found a list on the [npm site](https://www.npmjs.com/search?q=eslint-plugin). Any more convenient way you know of?

Thanks in advance!

**Comment by aladdin-add** [comment, marked as answer]
https://github.com/dustinspecker/awesome-eslint is another good place. :)

**Reply by 8ctopus** [reply]
thank you! so there's no complete repo besides npm?

**Reply by aladdin-add** [reply]
I guess so, these plugins were maintained by the community - we don't have a full list.
